src/client/wallet/PowerUpTransaction.js

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
 import { FormattedMessage, FormattedRelative, FormattedDate, FormattedTime } from 'react-intl';
 import BTooltip from '../components/BTooltip';
 
-const PowerUpTransaction = ({ timestamp, amount }) => (
+const PowerUpTransaction = ({ timestamp, amount, to, from }) => (
   <div className="UserWalletTransactions__transaction">
     <div className="UserWalletTransactions__icon-container">
       <i className="iconfont icon-flashlight_fill UserWalletTransactions__icon" />
     </div>
     <div className="UserWalletTransactions__content">
       <div className="UserWalletTransactions__content-recipient">
         <FormattedMessage id="powered_up" defaultMessage="Powered up " />
+        {to === from ? null : (
+          <Link to={`/@${to}`}>
+            &nbsp;<span className="username">{to}</span>
```

Comment:
I feel like we could have specific translation `power_up_to` with message `Power up {to}`, so it would work well with other languages as well.

Reply:
Will do.